### PR TITLE
Fix for SSR2 fixture not working locally 

### PR DESCRIPTION
**Pull request by kunalpanchal**
## Summary

- `/fixtures/ssr2` lacked a README.md on how to set things up locally, so I have added a Readme file similar to that of `/fixtures/ssr` since the steps are the same.
- Even after running the README steps I was not able to view the application in the browser and there were no error logs since the [Options type](https://github.com/facebook/react/blob/main/packages/react-dom/src/server/ReactDOMFizzServerNode.js#L35) does not have `onCompleteShell`.

## How did you test this change?

```shell

yarn && yarn build # To build for the latest react

cd fixtures/ssr2
yarn && yarn install
yarn start # Works well ✅
yarn start:prod # Works well ✅
```

### CODE REVIEW

File: fixtures/ssr2/server/render.js [hunk 1]
```diff
@@ -36,14 +38,16 @@ module.exports = function render(url, res) {
...
   let didError = false;
-  const data = createServerData();
   const {pipe, abort} = renderToPipeableStream(
     <DataProvider data={data}>
       <App assets={assets} />
     </DataProvider>,
     {
       bootstrapScripts: [assets['main.js']],
-      onCompleteShell() {
+      onAllReady(){
```
Comment (gaearon) [inline fixtures/ssr2/server/render.js:47]: this one is not required, right?

Comment (kunalpanchal) [reply fixtures/ssr2/server/render.js:47]: Nope, just added it for the ease of understanding all the various methods exposed and available. I could remove it, if required 👍

**Comment by gaearon** [comment]
thanks!

**Comment by kunalpanchal** [comment]
Thank you @gaearon for the quick review. Btw, this is my first contribution to react 🎉

**Pull request merged**